lib/firefox/webdriver/index.js

```diff
@@ -152,6 +152,19 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
     .getCapabilities()
     .set('pageLoadStrategy', get(options, 'pageLoadStrategy', 'normal'));
 
+  const android = firefoxConfig.android;
+  if (android) {
+    if (android.package) {
+      ffOptions.androidPackage(android.package);
+      if (android.activity) {
```

Comment:
What happens if no activity was provided?  Will this still run ok,  or do we need an else condition here?

Reply:
This is a good question regardless of the defaulting.  Eventually, `android.package` and `android.activity` get packaged up and sent back to the Web Driver implementation: either `chromedriver` or `geckodriver`.  Those implementations can choose what to do with the parameters.  `chromedriver` knows how to fill the activity corresponding to specific packages, and `geckodriver` will know how to do that too.  But in general it's better to be explicit.